### DISCUSSION TITLE
moko-widgets roadmap

### DISCUSSION THREAD

**Discussion by frichez** [icerockdev/moko-widgets#308] [Q&A]
Hello, 
Is moko-widgets still supported?  It has been a while since last update.
I can't use maven artifacts (0.1.0) using a brand new project created with android studio due to kotlin (native) incompatibility.
Would be nice to support latest Kotlin version.
Thks.

**Comment by Alex009** [comment]
hi. moko-widgets development was stoped in 2020. if you already have project with moko-widgets - we can update library in 2-3 weeks. but if you just want to start new project with moko-widgets - don't do this :)

**Comment by frichez** [comment]
Hi thks for the answer. It's too bad moko widgets is an excellent project! (like others you are/were? developing mojo-resources, mvvm, ..)  I was thinking to use it in a Multiplatform app that targets android/iOS and a proprietary native UI engine (JVM/jni) running on an POS device. Designing UI screens once, previewing the android rendering it in android studio and running on these different targets would be awesome.
